experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -159,7 +159,7 @@ export class SharedPropertyTree extends SharedObject {
 	}
 
 	private scopeFutureDeltasToPaths(paths?: string[]) {
-		const socket = (this.runtime.deltaManager as any).deltaManager.connection.socket;
+		const socket = (this.runtime.deltaManager as any).deltaManager.connectionManager.connection.socket;
```

Comment:
unrelated to this change, but it would be good to understand what's happening here, and if this is something the need more support, as this is busting the proxy, and digging deep into the connection

Reply:
created issue: #8446